CHANGELOG.adoc

```diff
@@ -5,6 +5,28 @@ https://spring.io/projects/spring-cloud-gcp[Spring Cloud GCP] is a set of integr
 This document provides a high-level overview of the changes introduced in Spring Cloud GCP by release.
 For a detailed view of what has changed, refer to the https://github.com/GoogleCloudPlatform/spring-cloud-gcp/commits/main[commit history] on GitHub.
 
+== 2.0.3 (2021-06-08)
+
+=== Cloud Pub/Sub
+* Exposed publisher endpoint for message ordering (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/421[#421])
+* Pub/Sub Health Indicator timeout increased to 2 seconds (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/420[#420]).
+* Gated Cloud Pub/Sub emulator autoconfiguration Pub/Sub module being present and enabled (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/446[#446])
+* `PubSubMessageHandler` now passes the original message to new success/failure callbacks, allowing applications to track message publish status (https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/482[#482]).
```

Comment:
Should we mention that `setPublishCallback()` is not deprecated?

Reply:
yep.